aperturedb/cli/configure.py

```diff
@@ -270,6 +270,61 @@ def activate(
         config_file.write(json.dumps(configs, indent=2, cls=ObjEncoder))
 
 
+@app.command()
+def remove(
+        name: Annotated[Optional[str], typer.Argument(
+            help="Name of this configuration to remove")],
+        remove_if_active: Annotated[bool, typer.Option(
```

Comment:
I'm not sure these parameters are worth the complexity.
If you want to activate another config first, we already have an action for that.
Automatically activating an arbitrary config seems like a recipe for confusion.

Reply:
this isn't activating another config first - by default it will fail to delete an active config.
the `remove-if-active` is kind of the `rm -rf` of 'just get rid of it'